Dockerfile.standalone

```diff
@@ -0,0 +1,28 @@
+FROM node:12 AS build
```

Comment:
are you planning on adding a dockerfile for the backend as well?

Reply:
@timja Yes it could be done if you think it's useful.

Reply:
yes it would be good imo.

Reply:
Ok, I'm on it.